### ISSUE TITLE
Output of id is of type string in Json-Api, but is outputted as original type.

### ISSUE THREAD

**Issue by Guidhouse** [joukevandermaas/saule#141]
```c#    
public class User
    {
        public long UserId { get; set; }//For reasons of the DBA's we cannot change this easily
        public string Description { get; set; }
    }
```
Using ApiResource with 'WithId("UserId")' and the JsonApiSerializer<UserResource> captures the Id for the schema.@default ( in Swashbuckler), but the value is outputted without the quotation marks mandated by Json-Api, as seen here: http://jsonapi.org/format/#document-resource-objects.

what we get is the following json:

```json
default": {
    "data": {
        "type": "user",
        "id": 1,
        "attributes": {
            "description": "Test"
        }, ...
```
I hope my explanation is clear enough, and if I get the time I will try and fix it myself and make a pull-request.

**Comment by joukevandermaas** [comment]
Good catch! A PR would be much appreciated.

**Comment by joukevandermaas** [comment]
Fixed in #149.
